docs/subnets/reference-cli-commands.md

```diff
@@ -0,0 +1,517 @@
+-g, --ledger                       use ledger instead of key (always true on mainnet, defaults to false on fuji)
+-l, --local                        deploy to a local network
+-m, --mainnet                      deploy to mainnet
+    --output-tx-path string        file path of the blockchain creation tx
+-s, --same-control-key             use creation key as control key
+    --subnet-auth-keys strings     control keys that will be used to authenticate chain creation
+-t, --testnet fuji                 deploy to testnet (alias to fuji)
+    --threshold uint32             required number of control key signatures to make subnet changes
+```
+
+### describe
+
+The subnet describe command prints the details of a subnet configuration
+to the console. By default, the command will print a summary of the
+configuration. By providing the --genesis flag, the command will instead
+print out the raw genesis file.
+
+**Usage:**
+
+`avalanche subnet describe [subnetName] [flags]`
+
+**Flags:**
+
+```
+-g, --genesis   Print the genesis to the console directly instead of the summary
+-h, --help      help for describe
+```
+
+### export
+
+The subnet export command prints the details of an existing subnet deploy.
+
+The command prompts for an output filename. You can also provide one with
+the --output flag.
+
+**Usage:**
+
+`avalanche subnet export [subnetName] [flags]`
+
+**Flags:**
+
+```
+-h, --help            help for export
+-o, --output string   write the export data to the provided file path
+```
+
+### import
+
+The subnet import command will import a subnet configuration from a file or a git repository.
+
+To import from a file, you can optionally provide the filepath as a command line argument.
+Alternatively, running the command without any arguments will trigger an interactive wizard.
+To import from a repo, go through the wizard. By default, an imported subnet will not
+overwrite an existing subnet with the same name. To allow overwrites, provide the --force
+flag.
+
+**Usage:**
+
+`avalanche subnet import [subnetPath] [flags]`
+
+**Flags:**
+
+```
+    --branch string   the repo branch to use if downloading a new repo
+-f, --force           overwrite the existing configuration if one exists
+-h, --help            help for import
+    --repo string     the repo to import (ex: ava-labs/avalanche-plugins-core) or url to download the repo from
+    --subnet string   the subnet configuration to import from the provided repo
+```
+
+### join
+
+The subnet join command configures your validator node to begin validating
+a new subnet.
+
+To complete this process, you must have access to the machine running your
+validator. If the CLI is running on the same machine as your validator,
+it can generate or update your node's config file automatically.
+Alternatively, the command can print the necessary instructions to
+update your node manually. To complete the validation process, the
+NodeID of your validator node must have been whitelisted by one of the
```

Comment:
Is it so? Don't we just whitelist subnet IDs, and the NodeID must be present in the AddValidator tx?

Reply:
Yes. If you don't you will just sync the chain, you won't actually be a validator. I updated the description.